### DISCUSSION TITLE
v0.0.6

### DISCUSSION THREAD

**Discussion by PeteRager** [PeteRager/lennoxs30#35] [Announcements]
Add support for turning ventilation mode on/off
Add support for turning allergenDefender on/off
Bug Fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/PeteRager/lennoxs30/releases/tag/v0.0.6'>v0.0.6</a>.</em>

**Comment by Koomafloo** [comment]
Tested on core-2021.7.4. Working well on S30 106151-01 running 03.81.0207 and Smart hub 105081-07 running 03.81.0769.

**Reply by PeteRager** [reply]
Good to hear!  Thanks for the update.
